source/network-security.txt

```diff
@@ -12,18 +12,137 @@ Network Security
    :depth: 1
    :class: onecol
 
-Intro statement
+By default, {+service+} secures your database deployments through the
+use of mandatory TLS/SSL connection encryption, and a VPC-first,
+minimum-access default policy. You can further configure these
+protections to meet your unique security needs and preferences.
 
-{+service+} Features and Best Practices for Network Security
-------------------------------------------------------------
+Use the foundational guidance on this page to plan the network
+security configuration for your {+clusters+}.
 
-Content here
+{+service+} Features and Recommendations for Network Security
```

Comment:
I believe Sarah wanted to strike Recommendations from these titles now. It is just "features for ABC".